commit d51be09f8ed659203ef36041d1c0553bbd21950b (refs/changes/82/2882/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-30T14:28:19-07:00 (1 year, 11 months ago)
    
    TOOLS-1909 urclient: release 1.2.0
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/Makefile b/Makefile
index 5738c51..ad26f07 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright 2015 Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 ROOT :=			$(PWD)
@@ -42,3 +42,22 @@ clean:
 	rm -f 0-npm-stamp
 	rm -rf node_modules
 
+.PHONY: cutarelease
+cutarelease:
+	[[ -z `git status --short` ]]  # If this fails, the working dir is dirty.
+	@which json 2>/dev/null 1>/dev/null && \
+	    ver=$(shell json -f package.json version) && \
+	    name=$(shell json -f package.json name) && \
+	    publishedVer=$(shell npm view -j $(shell json -f package.json name)@$(shell json -f package.json version) version 2>/dev/null) && \
+	    if [[ -n "$$publishedVer" ]]; then \
+		echo "error: $$name@$$ver is already published to npm"; \
+		exit 1; \
+	    fi && \
+	    echo "** Are you sure you want to tag and publish $$name@$$ver to npm?" && \
+	    echo "** Enter to continue, Ctrl+C to abort." && \
+	    read
+	ver=$(shell cat package.json | json version) && \
+	    date=$(shell date -u "+%Y-%m-%d") && \
+	    git tag -a "v$$ver" -m "version $$ver ($$date)" && \
+	    git push --tags origin && \
+	    npm publish
diff --git a/README.md b/README.md
index 1657cb1..e5d4ad2 100644
--- a/README.md
+++ b/README.md
@@ -1,14 +1,15 @@
 # node-urclient
 
+This repository is part of the Joyent Triton project. See the [contribution
+guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
+*Triton does not use GitHub PRs* -- and general documentation at the main
+[Triton project](https://github.com/joyent/triton) page.
+
 A high-level client for the use of the [ur
-agent](http://github.com/joyent/sdc-ur-agent) in Joyent SmartDataCenter.  The
-client allows the consumer to discover compute nodes (servers) in a
-SmartDataCenter deployment, copy files to and from those nodes, ping nodes, and
-run arbitrary commands.
-
-This repository is part of the Joyent SmartDataCenter project (SDC).  For
-contribution guidelines, issues, and general documentation, visit the main
-[SDC](http://github.com/joyent/sdc) project page.
+agent](http://github.com/joyent/sdc-ur-agent) in Joyent Triton DataCenter.  The
+client allows the consumer to discover compute nodes (servers) in a Triton
+DataCenter deployment, copy files to and from those nodes, ping nodes, and run
+arbitrary commands.
 
 ## Overview
 
diff --git a/package.json b/package.json
index 4a26e32..4f78d6d 100644
--- a/package.json
+++ b/package.json
@@ -1,8 +1,8 @@
 {
   "name": "urclient",
-  "version": "1.1.0",
-  "description": "ur client",
-  "author": "Joshua M. Clulow <jmc@joyent.com>",
+  "version": "1.2.0",
+  "description": "Triton ur client",
+  "author": "Joyent (joyent.com)",
   "dependencies": {
     "amqp": "0.2.4",
     "assert-plus": "0.2.0",
